src/Twig/MonitoringTwigExtension.php

```diff
@@ -78,7 +76,7 @@ public function embedMonitoring () : string
 
         return \sprintf(
             '<script src="%s"></script><script>window.TrackJS && TrackJS.install(%s)</script>',
-            $this->assetHelper->getUrl("@monitoring/js/trackjs.js"),
+            $this->assetProvider->getUrl("js/trackjs.js"),
```

Comment:
As this twig extension only has a single method, we can add the three lines to generate the URL inline here.